scene/main/viewport.h

```diff
@@ -626,6 +627,7 @@ class Viewport : public Node {
 	bool is_handling_input_locally() const;
 
 	bool gui_is_dragging() const;
+	bool gui_is_globally_dragging() const;
```

Comment:
`gui_is_dragging()` is exposed, while `gui_is_globally_dragging()` is not.
I wonder if the global drag couldn't replace the former one. Is there any use-case for the old method? Seems like it's just less reliable.

Reply:
Previously `gui_is_dragging()` could be used, to query, if a dragging action is active in the current Viewport.

With the changes of this PR drag-and-drop becomes an operation that spans multiple Viewports, so now there are two concepts:
1. Is drag-and-drop active, that started in this Viewport? (This PRs `gui_is_dragging()`)
2. Is drag-and-drop active, that can drop in the current Viewport? (This PRs `gui_is_globally_dragging()`)

For the setting, that there is only a single Viewport in the scene, both behave identical.

After giving it more thought, I believe the second concept is more important for users, So I will update the PR and simply replace `gui_is_dragging()` by the second concept.